packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -142,9 +142,6 @@ export default {
 
     const targetDatasetName = await (targetDataset ||
       promptForDatasetName(prompt, {message: 'Target dataset name:'}))
-    if (existingDatasets.includes(targetDatasetName)) {
-      throw new Error(`Target dataset "${targetDatasetName}" already exists`)
-    }
```

Comment:
Do we need to make any changes to the error logging?

Reply:
No, it gets caught in the same set of statements we already use.